<launch>
    <node pkg="move_base" type="move_base" name="move_base">
        <rosparam file="$(find eac_pkg)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find eac_pkg)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find eac_pkg)/params/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find eac_pkg)/params/local_costmap_params.yaml" command="load" />
        <param name="base_global_planner" value="global_planner/GlobalPlanner" />
        <param name="base_local_planner" value="teb_local_planner/TEBLocalPlannerROS" />
        <rosparam file="$(find eac_pkg)/params/teb_local_planner_params.yaml" command="load" />
        <param name="controller_frequency" value="10.0" />
    </node>
</launch>